country_update_demo/templates/update-record.html

```diff
@@ -1,55 +1,59 @@
 {% extends "govuk_template.html" %}
 {% block content %}
 
-<header>
-    <div class="row">
-        <div class="large-6 columns">
-            <h1 class="heading-medium" align="middle">Update country record</h1>
-        </div>
+<div class="grid-row">
+    <div class="column-two-thirds">
+        <h1 class="heading-large">
+            <span class="heading-secondary">Country Register</span>
+            Update this record
+        </h1>
+
+
+        <form method="post" action="/update-record/{{country}}">
+            <div class="form-group">
+                <label class="form-label-bold" for="country">country</label>
+                <input class="form-control" id="country" name="country" type="text" value="{{country}}" readonly>
```

Comment:
could this maybe be rendered just as page content rather than as an input field? (as suggested by @robinwhittleton)